src/routes/view-content/[guideId]/[bibleSection]/feedback/ResourceFeedbackModal.svelte

```diff
@@ -30,7 +32,7 @@
         [ContactType.Other]: $translate('page.feedback.contactType.other.value'),
     };
 
-    const contactTypeOptions = Object.values(ContactType).map((v) => ({
+    export const ContactTypeOptions = Object.values(ContactType).map((v) => ({
         value: v.toString(),
         label: contactTypeLabels[v],
     }));
```

Comment:
I don't think you can export from a `.svelte` file... Is this being used somewhere or can it be deleted?

Reply:
Yeah, this should've been cleaned up from where I was going a different direction at first.